scripts/zkai_attention_kv_d8_bounded_softmax_table_native_gate.py

```
@@ -0,0 +1,547 @@
+    }
+
+
+def mutation_cases_for(payload: dict[str, Any]) -> list[dict[str, Any]]:
+    if len(EXPECTED_MUTATION_NAMES) != EXPECTED_MUTATION_COUNT:
+        raise AttentionKvBoundedSoftmaxTableNativeGateError("internal mutation spec count drift")
+    cases = []
+    for name in EXPECTED_MUTATION_NAMES:
+        mutated = mutate_payload(payload, name)
+        try:
+            validate_payload(mutated, allow_missing_mutation_summary=True)
+        except Exception as err:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)